js/templates/components/pageControls.html

```diff
@@ -1,10 +1,12 @@
 <%
-  var countsAvailable = false;
```

Comment:
Can we make a seperate component for this style of page controls? I think the data required is quite different than the other style and I just am finding it confusing to blend the two.

Really, the only state the new style needs is `curPage` and `morePages: true/false`. I think it's important that the view is dumb about the curPage rather than implementing logic that forces a set page size. I could think of examples where a list gets new data added to the top of it where you'd want to let the first page grow without affecting the subsequent pages.